README.md

```diff
@@ -1 +1,65 @@
-First commit
+# JIRA bulk task creator
+
+## Install
+
+* Manage configuration in file [config.json](/Jira_tools/config.json)
+* Add permission to be executed: `chmod +x jira_projects_create_cards.py github_common.py`
```

Comment:
+x only on the file you exec.
In any case you can add the permission to the file already, then you can skip the manual step